examples/push/main.go

```diff
@@ -1,22 +1,20 @@
 package main
 
 import (
-	"fmt"
 	"os"
 
 	"gopkg.in/src-d/go-git.v4"
 	. "gopkg.in/src-d/go-git.v4/examples"
 )
 
 func main() {
-	CheckArgs("<repo path>")
-	repoPath := os.Args[1]
+	CheckArgs("<repository-path>")
```

Comment:
now it auto detects, what is being open and triggers a error, is documented on the method, I think is enough